It's better for us to see gcc output in the build log to ease debugging.
Quiet builds make that much harder.

Index: aiccu-20070115/unix-console/Makefile
===================================================================
--- aiccu-20070115.orig/unix-console/Makefile	2009-10-25 22:33:06.000000000 +0100
+++ aiccu-20070115/unix-console/Makefile	2009-10-25 22:33:12.000000000 +0100
@@ -26,7 +26,7 @@
 
 #CFLAGS	+= $(CWARNS) -D_GNU_SOURCE -D_DEBUG -g3 -O0
 CFLAGS	+= $(CWARNS) -D_GNU_SOURCE
-CC      = @gcc
+CC      = gcc
 RM      = rm
 
 # Add -O3 when nothing is specified yet
